scripts/run-front-playwright-container.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -e
+
+# Open the base osrd folder, assuming the script is located in osrd/scripts
+cd "$(dirname "$0")/.." 
```

Comment:
For consistency, either the other scripts (eg. `scripts/host-compose.sh`) should do this or this one shouldn't

Reply:
It is already done in sync-openapi.sh but not other scripts, so this needs to be settled before this pr is merged